plugins/redfish/fu-redfish-multipart-device.c

```diff
@@ -49,6 +49,20 @@ fu_redfish_multipart_device_get_parameters(FuRedfishMultipartDevice *self)
 	return g_steal_pointer(&str);
 }
 
+static size_t
+fu_redfish_multipart_device_location_headers_callback(char *ptr,
+						      size_t size,
+						      size_t nmemb,
+						      void *location)
+{
+	char **loc_str = (char **)location;
+	if ((size * nmemb) > 16 && g_ascii_strncasecmp(ptr, "Location:", 9) == 0) {
```

Comment:
Is this really case insensitive or can we use `g_str_has_prefix`?

Reply:
Headers are supposed to be case insensitive, but I don't think Curl does any processing on them, so I would suppose it is better to leave them like that to be safe